pkg/runctl/config_test.go

```diff
@@ -74,6 +74,10 @@ func Test_loadPipelineRunsConfig_EmptyEntries(t *testing.T) {
 	assert.DeepEqual(t, expectedConfig, resultConfig)
 }
 
+var metav1Duration = func(d time.Duration) *metav1.Duration {
+	return &metav1.Duration{Duration: time.Duration(d)}
+}
```

Comment:
Doesn't compile. Fix:
```suggestion
var metav1Duration = func(d time.Duration) *metav1.Duration {
	return &metav1.Duration{Duration: d}
}
```